.travis.yml

```diff
@@ -216,11 +216,12 @@ matrix:
           - /root/travis_bootstrap
           - /hab/studios/home--travis--build--habitat-sh
           - /root/.cargo
-      before_install:
+      script:
         - ./support/ci/fast_pass.sh || exit 0
+      before_deploy:
         - if [[ ! -x ./support/ci/deploy.sh ]]; then chmod +x ./support/ci/deploy.sh; fi
         - openssl aes-256-cbc -K $encrypted_50e90ce07941_key -iv $encrypted_50e90ce07941_iv -in ./support/ci/habitat-srv-admin.enc -out /tmp/habitat-srv-admin -d
-      script:
+      deploy:
         - sudo ./support/ci/deploy.sh
```

Comment:
I'm not clear on how this fixes the break. Given a successful build, the deploy phase will kick in and call `openssl` which I'd expect to break on an external PR. by moving the `openssl` call to deploy, won't it still be invoked on external PRs and blow up because of the secrets?

Have you tried adding a small change to sup or hab to get travis past the `fast_pass` check to validate?